aiven/resource_service_integration.go

```diff
@@ -471,6 +476,10 @@ func copyServiceIntegrationPropertiesFromAPIResponseToTerraform(
 	if err := d.Set("integration_type", integrationType); err != nil {
 		return err
 	}
+	integrationId := integration.ServiceIntegrationID
```

Comment:
Can we do without temporary `integrationId` variable?

Reply:
done